src/models/RegistrationSchema.js

```diff
@@ -61,6 +61,7 @@ function (Okta, BaseSchema, SchemaProperty) {
           this.properties.comparator = 'sortOrder';
           this.properties.sort();
         }
+        this.properties.policyId = resp.policyId;
```

Comment:
Isn't this the default policy Id ? Should we just call it `this.properties.defaultPolicyId` ? The difference is just that the server sends this instead of it being set the html. It would be nice if we can also add a small comment here for that.